src/libs/actions/PersonalDetails.js

```diff
@@ -66,20 +65,16 @@ function getDisplayName(login, personalDetail) {
 }
 
 /**
- * Returns object with first and last name errors. If either are valid,
- * those errors get returned as empty strings.
+ * Returns max character error text if true.
  *
- * @param {String} firstName
- * @param {String} lastName
- * @returns {Object}
+ * @param {Boolean} shouldReturnError
+ * @returns {String}
  */
-function getFirstAndLastNameErrors(firstName, lastName) {
-    return {
-        firstNameError: ValidationUtils.isValidLengthForFirstOrLastName(firstName) ? '' : Localize.translateLocal('personalDetails.error.firstNameLength'),
-        lastNameError: ValidationUtils.isValidLengthForFirstOrLastName(lastName) ? '' : Localize.translateLocal('personalDetails.error.lastNameLength'),
-    };
+function getMaxCharacterError(shouldReturnError) {
+    return shouldReturnError ? Localize.translateLocal('personalDetails.error.characterLimit', {limit: 50}) : '';
```

Comment:
I'm missing why `shouldReturnError` is necessary. Why not always return the error?

Reply:
Maybe the param name is not correct, should it simply be `error`?

Reply:
Okay I see now. I think the param name is not very clear. Can we stick with the more widely used `hasError` or `isError` please?